### PR TITLE
x264: drop ffmpeg support

### PR DESCRIPTION
**Pull request by iBobbyTS**
x264 doesn't require ffmpeg.

### CODE REVIEW

File: mingw-w64-x264/PKGBUILD [hunk 1]
```diff
@@ -74,7 +73,6 @@ package_x264() {
   # Note: libx264 is stricly not needed, but existing users of the x264 package
   # before the split expect the library to be included
```
Comment (lazka) [inline mingw-w64-x264/PKGBUILD:74]: imo this would break too many things

**Comment by iBobbyTS** [comment]
Adding ffmpeg while user only need x264 will cause too many extra libraries and potential problems. Is there a way for users to disable ffmpeg installation when installing x264 only on their case?

**Comment by lazka** [comment]
(ignore my previous comment, I mixed things up)

x264 currently uses ffmpeg, so the dependency is correct.

There is currently no way besides rebuilding.

**Comment by lazka** [comment]
Some examples in other distors:

* Debian uses ffmpeg: https://packages.debian.org/bookworm/x264
* Arch doesn't: https://archlinux.org/packages/extra/x86_64/x264/

I personally don't know enough about users of x264 to say what feature set of x264 they expect.

**Comment by filnet** [comment]
@iBobbyTS commit log should contain project name. Take a quick look at the commit history : https://github.com/msys2/MINGW-packages/commits/master

Actually, @lazka edited the PR title to match what the commit log should be.

**Comment by lazka** [comment]
If we can get any good reasons why installing ffmpeg is indeed a problem then we can talk about this. If not I'll close this.

**Comment by iBobbyTS** [comment]
When I build ffmpeg with a third party filter, I would success with libaom, libx265, but if I install x264 with pacman, which comes with a huge number of libraries, it would build fail.

**Comment by lazka** [comment]
So ffmepg fails to build for you is ffmpeg is installed already? How does it fail?

**Comment by lazka** [comment]
as I'm still not sure why exactly this would be needed, closing. If you can provide any more info feel free to create a new issue.

**Pull request closed without merging**